chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts

```diff
@@ -0,0 +1,91 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "com.chartboost.mediation.sdk.demo"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "com.chartboost.mediation.sdk.demo"
+        minSdk = 21
+        targetSdk = 34
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro",
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
+    }
+    kotlinOptions {
+        jvmTarget = "17"
+    }
+    buildFeatures {
+        compose = true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.5.7"
+    }
+    packaging {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+}
+
+dependencies {
+    // Chartboost Mediation SDK
+    implementation("com.chartboost:chartboost-mediation-sdk:4.9.0")
```

Comment:
you sure we sync'd the right branch?

Reply:
Yeah. Demo app update work will be done by @bwised.

Reply:
We don't need to merge this PR, but now that we know rsync can add the apps, 🤩 will push the script changes that were made to get this to run successfully.